src/eval/callstack.rs

```diff
@@ -219,6 +219,11 @@ impl CallStack {
         self.0.len()
     }
 
+    /// Return whether the callstack is empty.
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
```

Comment:
Clippy was complaining that this module has a `len` method without a `is_empty` method. @yannham said in a future PR we probably should change the visibility of many/all methods of this module to `pub(super)` but this is rather unrelated to this PR, so in the short term we add this function.